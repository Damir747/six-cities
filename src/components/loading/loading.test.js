import React from 'react';
import { render } from '@testing-library/react';
import { Router } from 'react-router-dom/cjs/react-router-dom.min';
import { createMemoryHistory } from 'history';
import Loading from './loading';

it(`Should Loading render correctly`, () => {
  const history = createMemoryHistory();
  const { getByText } = render(
    < Router history={history}>
      <Loading />
    </ Router>
  );
  const header = getByText(/Loading/i);
  expect(header).toMatchSnapshot();
});
