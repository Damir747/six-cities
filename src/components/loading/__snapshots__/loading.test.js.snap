// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Loading render correctly 1`] = `
<p>
  Loading...
</p>
`;
